### ISSUE TITLE
3/1 Code for Boston agenda

### ISSUE THREAD

**Issue by kevinrobinson** [studentinsights/studentinsights#135]
Trying this out.

#### Review priorities:
- current: [Student profile improvements](https://github.com/studentinsights/studentinsights/issues/5)
- next: [Productionize visualizations of academic progress](https://github.com/studentinsights/studentinsights/issues/2)
- after: [SPED referral form](https://github.com/studentinsights/studentinsights/issues/8)

#### Finish what's in progress:
+ [ ] merge PRs for student profile header with @BWieber (https://github.com/studentinsights/studentinsights/issues/93 etc.)
+ [ ] merge migrations for services with @BWieber (https://github.com/studentinsights/studentinsights/issues/115 etc.)
+ [ ] catch up with @erose on [newsfeed](https://github.com/studentinsights/studentinsights/issues/117)
+ [ ] make progress on [saving services](https://github.com/studentinsights/studentinsights/issues/115)

#### Help new folks pitch in:
- Talk about any questions from the project overview
- Help them set up development environment
- Start with [Help wanted issues](https://github.com/studentinsights/studentinsights/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
- Next, help with new pieces of current feature
- After, start with a new feature

cc @alexsoble

**Comment by alexsoble** [comment]
This looks like a fantastic flow!

**Comment by alexsoble** [comment]
How did it go at CFB?

**Comment by kevinrobinson** [comment]
This was a [great session](https://twitter.com/krob/status/704865050004168704)!

We made progress on the first two agenda items, but started a few other threads as well.
- @BWieber merged some backend work for [saving services](https://github.com/studentinsights/studentinsights/pull/132) and made more progress on [styling improvements to the student header](https://github.com/studentinsights/studentinsights/pull/108)
- @eudaimonious started on the SPED referral form feature and made an [initial PDF!](https://github.com/studentinsights/studentinsights/pull/137)
- @bw120 made great improvements on improving the styling to the [history of behavior incidents](https://github.com/studentinsights/studentinsights/pull/139)
- @sentientmachine made progress on improving the coloring of [lines in charts](https://github.com/studentinsights/studentinsights/issues/133)
- @lilybarrett got mostly up and running with her dev environment!
- @johnhrobb and Kemal gave some epic design feedback! capturing that in issues now :)

**Comment by kevinrobinson** [comment]
@therebelrobot hello over at https://github.com/sfbrigade/brigadehub/issues/46, I'm glad this is helpful!  Here are some thoughts, would love to hear any input you have. :)

We're trying to keep a flow shipping to users, and that requires full-stack work across product, design and engineering.  So far it's been easiest to get contributions on well-defined engineering work, since less context and ramp-up is necessary to contribute there.  Good UX and design work often takes a deeper shared understanding, so we've tried to give a first cut at that with a [product overview](https://github.com/studentinsights/studentinsights#product-overview) and demo site, and we focus on this rather than the tech stack when introducing the project.  Conversations with brand new folks are almost entirely about product and user stories rather than tech, and about communicating the feeling of regularly shipping small features.  But this takes time to coordinate, especially for new projects where shaping the user experience is so important.  It often takes work to discover clearly-defined engineering tasks.

Anyway, we'd love to learn from you and what works well as you try this out too! :)

**Comment by therebelrobot** [comment]
:+1: For sure! @judy5000 would definitely be interested in this flow, given her PM background, and as we're building out these initial pieces, demos and product overviews will for sure be handy once we hit beta. I agree that UX and design require a closer understanding, I may pull that from the agenda for now.

Judy, I would love to get your input on this, and see how we can adapt it to brigadehub's needs as we move forward. As this is more of a meta discussion, maybe we can start a new issue on brigadehub for Hack Night meeting structure?

**Comment by kevinrobinson** [comment]
:+1: cool, sounds great!
